### PR TITLE
IRIS-223 add ignore regions to cypress example

### PR DESCRIPTION
**Pull request by paweltomaszewskisaucelabs**

### CODE REVIEW

File: cypress/package-lock.json [hunk 1]
```diff
@@ -22,8 +22,9 @@
     },
     "node_modules/@ampproject/remapping": {
       "version": "2.2.1",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.1.tgz",
+      "resolved": "https://artifactory.tools.saucelabs.net:443/artifactory/api/npm/all-npm/@ampproject/remapping/-/remapping-2.2.1.tgz",
```
Comment (FriggaHel) [inline cypress/package-lock.json:25]: This must not be the internal registry.
Same applies to all that file.

File: cypress/package.json [hunk 2]
```diff
@@ -6,13 +6,15 @@
     "node": "18"
   },
   "scripts": {
-    "copy-locked": "cp ./cypress/e2e/saucedemo.locked.ts ./cypress/e2e/saucedemo.cy.ts",
```
Comment (FriggaHel) [inline cypress/package.json:9]: Why reorder ?
It was using alphabetical order for clarity.

**Comment by FriggaHel** [comment]
Can you add the documentation in the `README.md` please ?

File: cypress/package.json (hunk 2 above)
Comment (paweltomaszewskisaucelabs) [reply cypress/package.json:9]: I wanted to have an order how a user should run them. 

1. cypress
2. cypress-modified
3. cypress-ignored

File: cypress/package-lock.json (hunk 1 above)
Comment (paweltomaszewskisaucelabs) [reply cypress/package-lock.json]: fixed

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**